data/translations/en/register.json

```diff
@@ -1,8 +1,9 @@
 {
-  "h2-deadline": "Voter registration deadline in %state_name%",
+  "english_only": "",
   "heading": "Register to vote in %state_name%",
   "heading2": "How to register to vote in %state_name%",
   "meta_name": "Find out how to register to vote in %state_name%",
+  "selecciones_link": "",
```

Comment:
I think we need to understand this property better. It's very Spanish specific and I'm not sure if it will be a relevant field for all translations. Do you know of an example where it displays? We may need to remove this altogether of come up with a solution that isn't specific to the language.

Reply:
We need to confirm the logic and behavior of this option with Laura and then determine how we can make this not language specific.

Reply:
We will be updating this property key to `select_link`